store/tikv/2pc.go

```diff
@@ -99,10 +111,22 @@ func newTwoPhaseCommitter(txn *tikvTxn) (*twoPhaseCommitter, error) {
 				Op:  pb.Op_Lock,
 				Key: lockKey,
 			}
+			lockCnt++
 			keys = append(keys, lockKey)
 			size += len(lockKey)
 		}
 	}
+	if len(keys) > kv.TxnEntryCountLimit || size > kv.TxnTotalSizeLimit {
```

Comment:
Could this be large than TxnEntryCountLimit? When put key-value into TxnEntryCountLimit, we have already checked this.